tf-modules/cumulus/variables.tf

```diff
@@ -544,3 +544,11 @@ variable "ecs_custom_sg_ids" {
   type = list(string)
   default = []
 }
+
+variable "ecs_custom_sg_ids" {
```

Comment:
This appears to be a duplicate of the above variable with a updated default - the duplication should be removed and/or the update explained.